_posts/fr/newsletters/2022-10-05-newsletter.md

```diff
@@ -0,0 +1,208 @@
+      ce qui rend sa dépense rentable.
+
+    Aucun de ces éléments ne doit être réalisé en même temps que la mise en œuvre
+    du relais des transactions v3, mais plusieurs personnes ayant répondu au fil
+    de discussion semblaient être en faveur de tous les changements proposés.
+
+- **LN flow control:** Rene Pickhardt [a posté][pickhardt ml valve] à la liste de
+  diffusion Lightning-Dev un résumé de sa [recherche récente][pickhardt bitmex valve]
+  il a réalisé une étude sur l'utilisation du paramètre `htlc_maximum_msat` comme
+  valve de contrôle du débit. Comme [défini précédemment][bolt7 htlc_max] dans BOLT7,
+  `htlc_maximum_msat` est la valeur maximale qu'un nœud transmettra au prochain saut
+  dans un canal particulier pour une partie de paiement individuelle
+  ([HTLC][topic htlc]). 
+  Pickhardt aborde le problème d'un canal dans lequel plus de valeur circule dans une
+  direction que dans l'autre--ce qui finit par laisser le canal sans assez de fonds
+  pour transférer dans la direction surexploitée. Il suggère que le canal peut être
+  maintenu en équilibre en limitant la valeur maximale dans la direction surexploitée.
+  Par exemple, si un canal commence par autoriser le transfert de 1,000 sat dans les
+  deux sens, mais qu'il devient déséquilibré, essayez de réduire à 800 le montant
+  maximum par paiement transféré dans le sens surexploité. Les recherches de Pickhardt
+  fournissent plusieurs extraits de code qui peuvent être utilisés pour calculer les
+  valeurs `htlc_maximum_msat` appropriées. 
+
+    Dans un [autre courriel][pickhardt ratecards], Pickhardt suggère également
+    que l'idée précédente de *fee ratecards* (voir la [newsletter de la semaine dernière]
+    [news219 ratecards]) pourraient plutôt devenir des
+    *maximum amount per-forward ratecards*, où celui qui effectue la dépense se verrait facturer
+    un feerate plus bas pour envoyer de petits paiements et un feerate plus élevé
+    pour envoyer des paiements plus importants.  A la différence de la proposition
+    originale de ratecards, il s'agirait de montants absolus et non relatifs au
+    solde actuel du canal. Anthony Towns [a décrit][towns ratecards]
+    plusieurs défis avec l'idée originale des cartes de taux qui ne seraient pas
+    des problèmes pour le contrôle de flux basé sur l'ajustement de `htlc_maximum_msat`.
+
+    ZmnSCPxj [a critiqué][zmnscpxj valve] plusieurs aspects de l'idée, y compris
+    le fait que ceux qui effectuent la dépense pourraient toujours envoyer la même quantité de
+    valeur par le biais d'un canal à taux maximum inférieur, ce qui aurait pour
+    conséquence de le déséquilibrer à nouveau, simplement en divisant un paiement
+    global en petites parties supplémentaires. Towns a suggéré que ce problème
+    pourrait être résolu en limitant le taux.
+
+    La discussion semblait être en cours au moment où ce résumé a été écrit, mais
+    nous nous attendons à ce que plusieurs nouveaux aperçus viennent dans les
```

Comment:
Is this nous nous duplicate intentional?

Reply:
Yes, it's good (a reflexive verbe).

Reply:
Added a commit to bypass the duplicate word linter check